app/uk/gov/hmrc/tariffclassificationfrontend/views/template/activity_template.scala.html

```diff
@@ -0,0 +1,67 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.controllers.routes
+@import uk.gov.hmrc.tariffclassificationfrontend.models.Case
+@import uk.gov.hmrc.tariffclassificationfrontend.models.ReviewStatus.ReviewStatus
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.review_status_form
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.case_heading
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.tab_heading
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.print_tab
+@import uk.gov.hmrc.tariffclassificationfrontend.models.Event
+@import uk.gov.hmrc.tariffclassificationfrontend.models.Paged
+
+@import uk.gov.hmrc.tariffclassificationfrontend.utils.Dates
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.activity_event_format
+@import uk.gov.hmrc.tariffclassificationfrontend.models.Queue
+
+@import uk.gov.hmrc.tariffclassificationfrontend.models.request.AuthenticatedRequest
+
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{AppealStatus, AppealStatusChange, AssignmentChange, Case, CaseStatusChange, Event, EventType, ExtendedUseStatusChange, Note, Operator, Paged, Queue, QueueChange, ReviewStatus, ReviewStatusChange}
+@import uk.gov.hmrc.tariffclassificationfrontend.utils.Dates
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.{assignee, progressive_disclosure}
+@import uk.gov.hmrc.tariffclassificationfrontend.models.EventType
+
+@(c: Case, events: Paged[Event], queues: Seq[Queue])(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
+
+<table id="activity-events">
```

Comment:
Guessing you didnt fancy using the Json table generator thing for this?

Reply:
I guess It would be similar... we already have the components which solve the message content then... Maybe I prefer some sort of style